src/ImageRunModal.scss

```diff
@@ -0,0 +1,18 @@
+@import '@patternfly/patternfly/layouts/Grid/grid.css';
+
+// Limit the max-width more the override in podman.scss which is 40%
+#run-image-dialog-memory, #run-image-cpu-priority {
```

Comment:
It's better if we could use a common class for both of these, like`.run-image-limit` or something.

(See below for ID vs. class.)